enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Editor.tsx

```diff
@@ -12,9 +12,9 @@ import { getSetting } from "metabase/selectors/settings";
 import { ScrollArea } from "metabase/ui";
 import type Question from "metabase-lib/v1/Question";
 
-type NotebookProps = { onApply?: () => void };
+type EditorProps = { onApply?: () => void };
 
-export const Notebook = ({ onApply = () => {} }: NotebookProps) => {
+export const Editor = ({ onApply = () => {} }: EditorProps) => {
```

Comment:
We can also remove the alias from the `Notebook` component since the names don't clash anymore.
https://github.com/metabase/metabase/blob/ffdc4b2456fdba220cec33c120006f6497533621/enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Editor.tsx#L10